packages/system/src/Item/api/hooks/useItem/useItem.ts

```diff
@@ -1,25 +1,24 @@
+import { ItemFragment } from './../../operations.generated';
```

Comment:
Changes to use the code generated `ItemFragment` rather than the `Item` type and also use the co-located generated operations query rather than from common